idd/Energy+.idd.in

```diff
@@ -48608,6 +48608,61 @@ ZoneHVAC:AirDistributionUnit,
        \type object-list
        \object-list DesignSpecificationAirTerminalSizingName
 
+ZoneHVAC:ExhaustControl,
+       \memo Defines a controlled exhaust flow from a zone which feeds into 
+       \memo an AirloopHVAC:Exhaust system inlet.
+  A1 , \field Name
+       \required-field
+  A2 , \field Availability Schedule Name
+       \note Availability schedule name for this exhaust system. Schedule value > 0 means it is available.
+       \note If this field is blank, the exhaust system is always available. If the attached 
+       \note AirloopHVAC:ExhaustSystem is off, then the flow will be zero.
+       \type object-list
+       \object-list ScheduleNames
+  A3 , \field Zone Name
+       \note Zone the exhaust inlet node is part of
+       \type object-list
+       \object-list ZoneNames
+  A4 , \field Inlet Node Name
+       \note Inlet node name for the exhaust
+       \required-field
+       \type node
+  A5 , \field Outlet Node Name
+       \note Outlet node name for the exhaust
+       \required-field
+       \type node
+  N1 , \field Design Exhaust Flow Rate 
+       \autosizable
+       \units m3/s
+       \minimum> 0
+       \default autosize
+  A6 , \field Flow Control Type
+       \note Control type of the exhaust fan
+       \type choice
+       \key Scheduled
+       \key FollowSupply
+       \default Scheduled
+  A7 , \field Exhaust Flow Fraction Schedule Name
+       \note Schedule name of the exhaust flow fraction. Used only with Scheduled control type.
+       \note If this field is blank, the flow fraction is always 1.0.
+       \type object-list
+       \object-list ScheduleNames
+  A8 , \field Supply Node or NodeList Name
+       \note Used only with FollowSupply control type.
+       \type node
+  A9,  \field Minimum Zone Temperature Limit Schedule Name
+       \note Schedule name of the Minimum Zone Temperature Limit
```

Comment:
\note Schedule name . . . . . Limit in deg. C
\note If this field is blank, there is no limit.

Reply:
Text modified.